docs/styleguide.md

```diff
@@ -0,0 +1,31 @@
+# ESL Code Styleguide
+
+## Naming convention agreements
+
+- `_private` (private notation) - **mast use**
+  Typical cases: 
+  - For inner details that should be hidden from outside world. 
```

Comment:
extra point

And I think it would be more appropriate to start with a lowercase letter.